selectrum.el

```diff
@@ -936,16 +924,19 @@ candidates a bit better (in particular you can immediately press
 directory). For PROMPT, DIR, DEFAULT-DIRNAME, MUSTMATCH, and
 INITIAL, see `read-directory-name'."
   (let ((dir (expand-file-name (or dir default-directory)))
-        (default (or default-dirname initial dir)))
-    (setq default (directory-file-name dir))
+        (default (directory-file-name (or default-dirname initial dir))))
     ;; Elisp way of getting the parent directory. If we get nil, that
     ;; means the default was a relative path with only one component,
     ;; so the parent directory is dir.
     (setq dir (or (file-name-directory
                    (directory-file-name default))
                   dir))
     (selectrum-read-file-name
-     prompt dir default mustmatch nil #'file-directory-p)))
+     prompt dir
+     ;; show current dir first
+     (file-name-as-directory
+      (file-name-nondirectory default))
+     mustmatch nil #'file-directory-p)))
```

Comment:
What does this change do?

Reply:
This sorts the directory first, this way you can immediately press RET to select the directory you are currently in. Before it would insert the dir name without slash but I think this is a little bit more convenient. Completion starts at parent directory with the current dir sorted at top.

Reply:
Ok, yeah, that seems better. Thanks.